src/utilities/getVTokenByAddress.ts

```diff
@@ -1,10 +1,17 @@
+import config from 'config';
 import { VToken } from 'types';
 
-import { VBEP_TOKENS } from 'constants/tokens';
+import { MAINNET_VBEP_TOKENS, TESTNET_VBEP_TOKENS, VBEP_TOKENS } from 'constants/tokens';
 
-const getVTokenByAddress = (address: string) =>
-  address.toLowerCase() in VBEP_TOKENS
+const getVTokenByAddress = (address?: string | null) => {
+  // get the vBNB of the current network to serve as a fallback
+  const vBnb = config.isOnTestnet
+    ? TESTNET_VBEP_TOKENS['0x2e7222e51c0f6e98610a1543aa3836e092cde62c']
+    : MAINNET_VBEP_TOKENS['0xa07c5b74c9b40447a954e1466938b865b6bbea36'];
```

Comment:
I wouldn't fallback to vBNB, as I see two issues with that:
- vBNB only exists in the main pool right now, but technically we could have a vBNB token in an isolated pool in which case defaulting to the main pool token could be misleading
- it will cause issue once we switch to being a dApp multichain

I'd return `undefined` when address is `null` instead. Let me know your thoughts

Reply:
Also in the specific case for transactions there is this

```
  const token =
    getTokenByAddress(tokenAddress) ||
    getVTokenByAddress(tokenAddress)?.underlyingToken ||
    TOKENS.xvs;
```

Which covers the undefined case